fitbenchmarking/input_parsing.py

```diff
@@ -195,19 +124,103 @@ def parse_nist_file_line_by_line(lines):
             residual_sum_sq = float(line.split()[4])
 
         elif line.startswith('Data:'):
+
             if 0 == data_idx:
                 data_idx += 1
             elif 1 == data_idx:
                 data_pattern_text = lines[idx:]
                 idx = len(lines)
             else:
                 raise RuntimeError('Error parsing data line: {}'.format(line))
+
         else:
-            print("unknown line in supposedly NIST test file, ignoring: {0}".format(line))
+            ignored_lines += 1
+            # print("unknown line in supposedly NIST test file, ignoring: {0}".format(line))
```

Comment:
Yes, it gives out a lot of information that is not particularly useful and makes the output harder to understand overall. The user gets a notification with how many lines were ignored and what to do to see them explicitly.

Reply:
just delete this and we are good to go

Reply:
Sorry, I did not understand it in the first place. Just to make sure, are you referring to the comment?

Reply:
Sorry, did not understand what you were referring to in the first place. Just to make sure, do you want to delete the comment? Because in that case, I would argue to let the comment in.

Reply:
yes, the commented out line. I assume it is left in to help with debugging, but is not much use otherwise. So now we know it works there is no need for it. Unless I have missed something

Reply:
In case the user wants to see the output explicitly, he can follow the instructions in the terminal output and come to this line to uncomment it. I left it like that instead of asking the user for input directly. Should I change it to the latter?

Reply:
ah ok. Make sure this is added to the doc too from issue #25